src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphEntityDAO.scala

```diff
@@ -100,6 +107,9 @@ class GraphEntityDAO extends EntityDAO with GraphDAO {
   }
 
   override def cloneEntities(destWorkspaceContext: WorkspaceContext, entities: Seq[Entity], txn: RawlsTransaction) = txn withGraph { db =>
-    entities.map( e => save(destWorkspaceContext, e, txn) )
+    //save the entities with no references to other entities first
```

Comment:
Compare to [here](https://github.com/broadinstitute/rawls/pull/114/files#diff-f49c3687908d541f0fd60b9be8c20db4R104).